boa/src/builtins/math/mod.rs

```diff
@@ -134,12 +134,10 @@ impl Math {
     /// [spec]: https://tc39.es/ecma262/#sec-math.atan2
     /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2
     pub(crate) fn atan2(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
-        Ok(Value::from(if args.is_empty() {
-            f64::NAN
-        } else {
-            f64::from(args.get(0).expect("Could not get argument"))
-                .atan2(args.get(1).expect("Could not get argument").to_number())
-        }))
+        match (args.get(0), args.get(1)) {
+            (Some(y), Some(x)) => Ok(Value::from(f64::from(y).atan2(f64::from(x)))),
+            _ => Ok(Value::from(f64::NAN)),
+        }
```

Comment:
```suggestion
        Ok(match (args.get(0), args.get(1)) {
            (Some(y), Some(x)) => f64::from(y).atan2(f64::from(x)),
            _ => f64::NAN,
        }.into())
```
I think it would be clearer if we take the `Ok(Value::from(...))` and move it to the match level.

Reply:
:heavy_check_mark: